src/sftp/handler.go

```diff
@@ -209,23 +209,32 @@ func (receiver *SftpHandler) CopyFiles() {
 // copySingleFile moves a single file from an external SFTP server to our blob storage. Zip files go to an `unzip`
 // folder and then we call the zipHandler.Unzip. Other files go to `import` to begin processing
 func (receiver *SftpHandler) copySingleFile(fileInfo os.FileInfo, index int, directory string) {
-	slog.Info("Considering file", slog.String("name", fileInfo.Name()), slog.Int("number", index))
+	slog.Info("Considering file", slog.String(utils.FileNameKey, fileInfo.Name()), slog.Int("number", index))
 	if fileInfo.IsDir() {
 		slog.Info("Skipping directory", slog.String(utils.FileNameKey, fileInfo.Name()))
 		return
 	}
 
-	file, err := receiver.sftpClient.Open(directory + "/" + fileInfo.Name())
+	fullFilePath := directory + "/" + fileInfo.Name()
+
+	fileReadCloser, err := receiver.sftpClient.Open(fullFilePath)
```

Comment:
This seems like a weird name compared to original. Is it not the 'file' anymore?